### PR TITLE
Demo for UDT graph support

### PR DESCRIPTION
**Pull request by JoshEngels**

### CODE REVIEW

File: GraphNodeClassification.ipynb [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+    ")"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Evaluation\n",
+    "We can evaluate the performance of UDT by calculating the Area Under the Receiver Operating Characteristic Curve (ROC AUC). UDT achieves about 91% with the default configurations in this notebook, and as noted above we achieve 93-94% with longer training time (the previous SOTA was [90%](https://paperswithcode.com/sota/node-classification-on-yelpchi)). Here, we show the predict API, where we pass in samples without labels."
```
Comment (dtr3) [inline GraphNodeClassification.ipynb:130]: maybe we can just say ROC AUC

File: GraphNodeClassification.ipynb [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+    "        **{f\"col_{i}\": bolt.types.numerical([0, 1]) for i in range(32)},\n",
+    "        \"target\": bolt.types.categorical(),\n",
+    "        \"neighbors\": bolt.types.neighbors(),\n",
+    "    },\n",
+    "    target=\"target\",\n",
+    "    n_target_classes=2,\n",
+    "    integer_target=True,\n",
+    "    # Optional: you can remove the next line to increase AUC to ~0.93 at the\n",
+    "    # cost of increasing training time\n",
+    "    options={\"contextual_columns\": False},\n",
```
Comment (dtr3) [inline GraphNodeClassification.ipynb:82]: contextual columns is true by default?

**Review by dtr3 — APPROVED** [review]
LGTM

File: GraphNodeClassification.ipynb (hunk 2 above)
Comment (JoshEngels) [reply GraphNodeClassification.ipynb:82]: I think you're right, this is by default false

**Pull request merged**